### PR TITLE
new behavior key support added for hyperlinks with openNewTab key

### PR DESCRIPTION
**Pull request by FarhanShoukat**

### CODE REVIEW

File: delta/html.py [hunk 1]
```diff
@@ -258,11 +265,19 @@ def classes_check(op):
     return True
 
 
+def _open_new_tab(context: dict):
+    if 'behavior' in context:
+        if context['behavior'] == LinkBehavior.NEW_TAB:
+            return True
```
Comment (ruhaib) [inline delta/html.py]: can we do `return context['behavior'] == LinkBehavior.NEW_TAB` ?

**Review by ruhaib — APPROVED** [review]
Small suggestion, if possible.

File: delta/html.py [hunk 2]
```diff
@@ -258,11 +265,18 @@ def classes_check(op):
     return True
 
 
+def _open_new_tab(context: dict):
+    if 'behavior' in context:
+        return context['behavior'] == LinkBehavior.NEW_TAB
+    else:
+        return context.get('openNewTab')
```
Comment (stevenmoseley) [inline delta/html.py:272]: Could this work?
```
return context.get('behavior') == LinkBehavior.NEW_TAB or context.get('openNewTab')
```

**Review by stevenmoseley — APPROVED** [review]

File: delta/html.py (hunk 2 above)
Comment (FarhanShoukat) [reply delta/html.py:272]: No. `behavior` can have other values. In that case, it will fall back to `openNewTab` key, which is not the expected behavior.

**Pull request merged**